MpsAllocatorSample/Program.cs

```diff
@@ -203,6 +203,24 @@ static async Task RequestMultiplayerServer()
             string region = Console.ReadLine();
             req2.PreferredRegions = new List<string>() {region};
             req2.SessionId = Guid.NewGuid().ToString();
+            // InitialPlayers is the list of the initial players of the game
+            // they might or might not end up connecting to the game
+            Console.WriteLine("Type initial players ID, Enter to confirm. Empty string when you're done");
+            string id = Console.ReadLine();
```

Comment:
I am guessing this is the general format you are using for input (alternatively you can take a comma separate values in one line and then split it)

Reply:
yeah, so far we've been asking the user to input stuff (like Build) and press Enter to confirm